_docs/installing/debian.md

```diff
@@ -18,16 +18,16 @@ sudo apt-get install git libopus-dev libffi-dev libsodium-dev ffmpeg -y
 sudo apt-get install build-essential libncursesw5-dev libgdbm-dev libc6-dev zlib1g-dev libsqlite3-dev tk-dev libssl-dev openssl -y
 
 # If using Debian Stretch or lower, you need to install Python too using...
-sudo apt-get install python3.5 python3-pip -y
+sudo apt-get install python3.8 python3-pip -y
```

Comment:
I'm not sure that Python 3.8 is readily available in the Debian default repos. Can you confirm? If it isn't, we'll have to build 3.8 from source instead.